test/MatrixProductOperator_test.jl

```diff
@@ -91,6 +105,21 @@
                 arrays = [rand(1, 2, 2), rand(1, 1, 2, 2), rand(1, 2, 2)]
                 MatrixProductOperator{Closed}(arrays) isa TensorNetwork{MatrixProductOperator{Closed}}
             end
+
+            @testset "`Tensor.meta[:alias]`" begin
```

Comment:
Why 2 test sets with the same name?

Reply:
One is in the `testset` "`Closed` boundary" and the other one is in "`Open` boundary`, should I name them differently?

Reply:
Now is much better. But this `alias` test set into a `Metadata` test set. There might be other metadata that we want to check in the future.